controller/network/network.go

```diff
@@ -95,7 +98,7 @@ type Network struct {
 	serviceMisconfiguredTerminatorCounter     metrics.IntervalCounter
 }
 
```

Comment:
The network already has the raft controller. It's available as the command.Dispatcher instance. I think in this case it would make sense for the RaftRouter to get that and if it's not nil, try to cast it to `*raft.Controller`. Does that make sense?

Reply:
Makes sense to me after that explanation!